sql/information-schema.md

```diff
@@ -0,0 +1,441 @@
+           TABLE_CATALOG: def
+            TABLE_SCHEMA: test
+              TABLE_NAME: t1
+             COLUMN_NAME: a
+        ORDINAL_POSITION: 1
+          COLUMN_DEFAULT: NULL
+             IS_NULLABLE: YES
+               DATA_TYPE: int
+CHARACTER_MAXIMUM_LENGTH: NULL
+  CHARACTER_OCTET_LENGTH: NULL
+       NUMERIC_PRECISION: 11
+           NUMERIC_SCALE: 0
+      DATETIME_PRECISION: NULL
+      CHARACTER_SET_NAME: NULL
+          COLLATION_NAME: NULL
+             COLUMN_TYPE: int(11)
+              COLUMN_KEY: 
+                   EXTRA: 
+              PRIVILEGES: select,insert,update,references
+          COLUMN_COMMENT: 
+   GENERATION_EXPRESSION: 
+1 row in set (0.01 sec)
+```
+
+The corresponding `SHOW` statement is as follows:
+
+```sql
+mysql> SHOW COLUMNS FROM t1 FROM test;
++-------+---------+------+------+---------+-------+
+| Field | Type    | Null | Key  | Default | Extra |
++-------+---------+------+------+---------+-------+
+| a     | int(11) | YES  |      | NULL    |       |
++-------+---------+------+------+---------+-------+
+1 row in set (0.00 sec)
+```
+
+## ENGINES Table
+
+`ENGINES` 表提供了关于存储引擎的信息。从和 MySQL 兼容性上考虑，TiDB 一直将 InnoDB 描述为唯一支持的引擎。
+
+```sql
+mysql> SELECT * FROM engines\G
+*************************** 1. row ***************************
+      ENGINE: InnoDB
+     SUPPORT: DEFAULT
+     COMMENT: Supports transactions, row-level locking, and foreign keys
+TRANSACTIONS: YES
+          XA: YES
+  SAVEPOINTS: YES
+1 row in set (0.00 sec)
+```
+
+## KEY\_COLUMN\_USAGE Table
+
+`KEY_COLUMN_USAGE` 这张表描述了列的键约束，比如主键约束。
+
+```sql
+mysql> SELECT * FROM key_column_usage WHERE table_schema='mysql' and table_name='user'\G
+*************************** 1. row ***************************
+           CONSTRAINT_CATALOG: def
+            CONSTRAINT_SCHEMA: mysql
+              CONSTRAINT_NAME: PRIMARY
+                TABLE_CATALOG: def
+                 TABLE_SCHEMA: mysql
+                   TABLE_NAME: user
+                  COLUMN_NAME: Host
+             ORDINAL_POSITION: 1
+POSITION_IN_UNIQUE_CONSTRAINT: NULL
+      REFERENCED_TABLE_SCHEMA: NULL
+        REFERENCED_TABLE_NAME: NULL
+       REFERENCED_COLUMN_NAME: NULL
+*************************** 2. row ***************************
+           CONSTRAINT_CATALOG: def
+            CONSTRAINT_SCHEMA: mysql
+              CONSTRAINT_NAME: PRIMARY
+                TABLE_CATALOG: def
+                 TABLE_SCHEMA: mysql
+                   TABLE_NAME: user
+                  COLUMN_NAME: User
+             ORDINAL_POSITION: 2
+POSITION_IN_UNIQUE_CONSTRAINT: NULL
+      REFERENCED_TABLE_SCHEMA: NULL
+        REFERENCED_TABLE_NAME: NULL
+       REFERENCED_COLUMN_NAME: NULL
+2 rows in set (0.00 sec)
+```
+
+## SCHEMATA Table
+
+SCHEMATA 表提供了关于数据库的信息。表中的内容和 `SHOW DATABASES` 语句执行结果基本等价。
+
+```sql
+mysql> select * from SCHEMATA;
++--------------|--------------------|----------------------------|------------------------|----------+
```

Comment:
The `DEFAULT_CHARACTER_SET_NAME` and `DEFAULT_COLLATION_NAME` are not the same as that in [the English version](https://github.com/pingcap/docs/pull/994/files). Please help check which one is more accurate.

Reply:
@tiancaiamao PTAL

Reply:
This is the accurate version. ``` 
mysql> SELECT * FROM schemata;
+--------------+--------------------+----------------------------+------------------------+----------+
| CATALOG_NAME | SCHEMA_NAME        | DEFAULT_CHARACTER_SET_NAME | DEFAULT_COLLATION_NAME | SQL_PATH |
+--------------+--------------------+----------------------------+------------------------+----------+
| def          | INFORMATION_SCHEMA | utf8mb4                    | utf8mb4_bin            | NULL     |
| def          | mynewdb            | utf8mb4                    | utf8mb4_bin            | NULL     |
| def          | mysql              | utf8mb4                    | utf8mb4_bin            | NULL     |
| def          | PERFORMANCE_SCHEMA | utf8mb4                    | utf8mb4_bin            | NULL     |
| def          | test               | utf8mb4                    | utf8mb4_bin            | NULL     |
+--------------+--------------------+----------------------------+------------------------+----------+
5 rows in set (0.00 sec)```